posts/2021/weather-station-data-preprocess.ipynb

```diff
@@ -0,0 +1,1934 @@
+{
```

Comment:
<p><code>pandas.datetime</code>  -&gt; <code>pandas.to_datetime</code></p><p> </p><p>We could also link to the documentation page: https://pandas.pydata.org/docs/reference/api/pandas.to_datetime.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds/pull/134/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>